composer.json

```diff
@@ -38,12 +38,13 @@
         "laminas/laminas-coding-standard": "~2.0.0",
         "laminas/laminas-mvc": "^3.1.1",
         "laminas/laminas-servicemanager": "^3.4",
+        "mikey179/vfsstream": "2.0.x-dev",
         "phpunit/phpunit": "^9.4.1",
         "vimeo/psalm": "^4.4.1"
     },
     "autoload": {
         "files": [
-            "src/autoload.php"
+            "autoload/autoload.php"
```

Comment:
minor: not sure why this change - if it's just consistency with other repos it's fine, but in general it should be separate PR...

Reply:
Can move that to another PR if necessary. 
I personally just dont like autoload files being part of the "src" file as these do not follow any namespaces, e.g.

Imho, `src/` is meant to contain sources like interfaces, classes, traits, functions, e.g. So files containing `function` would be okay but files which do only provide autoloading polyfills should not be part of the `src/`.

I'm fine with keeping it as is aswell, just found it annoying.

Reply:
Keeping it under `src/` keeps it consistent with our other repos. If you want to revisit that and suggest an alternative location that _all_ repos use, let's add that to the TSC agenda.

Reply:
@weierophinney which repositories do you mean?
https://github.com/laminas/laminas-cache/blob/2.12.x/autoload/patternPluginManagerPolyfill.php
https://github.com/laminas/laminas-form/blob/2.15.x/autoload/formElementManagerPolyfill.php

The only project I know besides of these is https://github.com/laminas/laminas-zendframework-bridge which has the `autoload.php` under `source`. 🤷🏼‍♂️ 

Anyways, I've reverted that change as I dont see any value in these discussions and neither have a too strong opinion on that. I am fine with "keep things as is", even tho they're not consistent.